src/CMakeLists.txt

```diff
@@ -27,6 +27,11 @@ if (WIN32)
 
 elseif (UNIX)
   add_executable(${EXECUTABLE_TITLE} ${SOURCE_FILES})
+  if(WITH_UCHECKMARKS)
```

Comment:
Please refactor this as `if (NOT UCHECKMARKS)`

Reply:
if(NOT <condition>) says CMake documentation
https://cmake.org/cmake/help/latest/command/if.html